text/0000-editions.md

```diff
@@ -0,0 +1,86 @@
+- Start Date: 2018-09-04
+- RFC PR:
+- Ember Issue:
+
+# Editions
+
+## Summary
+
+Introduce a new concept: editions. Every few years, we will declare a new edition of Ember that bundles up accumulated incremental improvements into a cohesive package.
+
+Editions give us an opportunity to bring our documentation and marketing up-to-date to reflect the improvements we’ve made since the previous edition. They are also the time to synchronize Ember and the wider ecosystem, including updating default settings, blueprints, addons, and more.
+
+Editions also help ensure that Ember users are getting a cohesive, polished experience. The right time to declare a new edition is when:
+
+
+- A significant, coherent set of new features and APIs have all landed in the stable channel.
+- Error messages and the developer ergonomics of those new features have been fully polished.
+- Tooling (the Ember Inspector, blueprints, codemods, etc.) has been updated to work with these new features.
+- API documentation, guides, tutorials, and example code has been updated to incorporate the new features.
+
+## Motivation
+
+Ember releases a new, stable version every six weeks. For existing users, this drumbeat of incremental improvement is easier to keep up with than splashy, big-bang releases.
+
+However, this steady stream of small improvements comes with some downsides.
+
+- For Ember users, it’s hard to keep track of evolving best practices, and understand how new features are supposed to work together.
+- For non-Ember users, it’s not obvious what has changed and whether Ember is worth another look.
+- For Ember contributors, it’s not clear when to update documentation and marketing, because there’s always more features in the pipeline.
+
+This RFC proposes a new release concept called “editions”, which provides a lens through which to view these accumulated changes. It also marks an opportunity to synchronize the wider ecosystem with the latest changes to Ember, including updates to defaults, documentation, addons, and more.
+
+## Detailed design
+
+Editions touch several parts of the Ember ecosystem, and generally tie together a set of incremental changes with a unifying theme. Most of these changes should come from existing community processes (i.e. RFCs), with editions serving as a higher level conceptual grouping.
+
+### Editions vs. Releases
+Editions are not separate releases, but rather a specific range of releases belong to an edition. The current edition will be open-ended - it’s terminating release is determined by whenever the next edition is declared.
```

Comment:
Can we take this to mean that editions will not overlap?

Reply:
Yep!